src/lib/hasExample.ts

```
@@ -1,4 +1,15 @@
+import { useTheme } from '../composables/useTheme'
+import { formatJson } from './formatJson'
+
 export function hasExample(schema: any, visited: Set<any> = new Set(), level: number = 0): boolean {
+  if (!schema) {
+    return false
+  }
+
+  if (useTheme().getSpecConfig().avoidCirculars.value) {
+    schema = JSON.parse(formatJson(schema))
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)